.travis.yml

```diff
@@ -1,23 +1,11 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
   - hhvm
-  - 7
 
 env:
   - VARNISH_VERSION=4.0
 
-matrix:
-  include:
-    - php: 5.5
-      env: VARNISH_VERSION=3.0
-    - php: 5.3
-      env: SYMFONY_VERSION=2.3.* VARNISH_VERSION=4.0 COMPOSER_FLAGS="--prefer-lowest"
-
```

Comment:
changes in this file need to be reverted, just to have faster cycles with travis.